polkadot-protocol/architecture/polkadot-chain/overview.md

```diff
@@ -1,4 +1,260 @@
 ---
 title: Overview
-description: TODO
----
\ No newline at end of file
+description: Explore the fundamental concepts of the Polkadot protocol, including account addresses, extrinsics, events and runtime upgrades.
+---
+
+# Overview
+
+Since the release of Bitcoin in 2009, blockchain projects increased exponentially to the order of tens of thousands. 
```

Comment:
I'm completely sure we should leave this.. I thin starting starting with the next paragraph should suffice.
```suggestion
Since the release of Bitcoin in 2009, the amount of blockchain projects increased to the order of tens of thousands. 
```